be/src/exprs/function_context.cpp

```diff
@@ -22,6 +22,7 @@
 #include "column/type_traits.h"
 #include "exprs/agg/java_udaf_function.h"
 #include "runtime/runtime_state.h"
+#include "storage/rowset/bloom_filter.h"
```

Comment:
unused?

Reply:
FunctionContext::~FunctionContext() = default; need to know NgramBloomFilterState's complete definition which is in bloom_filter.h